packages/dappmanager/src/calls/sensorsDataGet.ts

```diff
@@ -0,0 +1,9 @@
+import { Sensors } from "@dappnode/common";
+import { sensors } from "@dappnode/hostscriptsservices";
+
+/**
+ * Returns the sensors info of the host
+ */
+export async function sensorsDataGet(): Promise<Sensors> {
+    return await sensors();
```

Comment:
`sensors` is not verbose at all and functions/methods should include verbs, like `getCpuTemperature`